matterpy/manager.py

```diff
@@ -31,13 +32,20 @@ def load_plugin(self, name, plugin_conf):
             print("Error during module init: %s" % str(exc))
 
     def register(self, plugin):
-        self.plugins.append(plugin)
+        "Deprecated! Alias for register_"
```

Comment:
This is a bit odd... not really a docstring but a string not assigned to a var.

Anyhow shouldn't this not be a `DeprecationWarning` if it is really a deprecation?